data/scenarios/Challenges/2048.yaml

```diff
@@ -0,0 +1,188 @@
+name: "2048"
+description: Make 2048!
+objectives:
+  - goal:
+    - OK, it's not really the same as the classic "2048" game.
+      However, your goal is still to make 2048.
+    condition: |
+      try {
+        as base {has "2048"}
+      } { return false }
+solution: |
+  def makeN : int -> cmd () = \n.
+    if (n == 1)
+      {harvest; return ()}
+      {makeN (n/2); makeN (n/2); make (format n)}
+  end;
+  place "1";
+  makeN 2048
+entities:
+  - name: "1"
+    display:
+      attr: gold
+      char: "1"
+    description:
+    - This is a one.  Maybe you can combine it with other ones
+      somehow.
+    properties: [growable, portable, infinite]
+  - name: "2"
+    display:
+      attr: gold
+      char: "2"
+    description:
+    - This is a two.  Maybe you can combine it with other twos
+      somehow.
+    properties: [portable]
+  - name: "4"
+    display:
+      attr: gold
+      char: "4"
+    description:
+    - This is a four.  You get the idea.
+    properties: [portable]
+  - name: "8"
+    display:
+      attr: gold
+      char: "8"
+    description:
+    - An eight.
+    properties: [portable]
+  - name: "16"
+    display:
+      attr: gold
+      char: "#"
```

Comment:
How about using powers of two instead? :slightly_smiling_face: So 1 is 0, 2 is 1, 4 is 2,...

Reply:
Oh, good idea.  Using hex digits, of course, so 2048 will look like `b`.

Reply:
So I tried that and it turned out to just be super confusing when you see them in the inventory.  The `char` attributes are never going to be seen or matter anyway, they just show up in the inventory.  So instead I changed them all to `#`.